packages/admin/cms-admin/src/dam/DataGrid/selection/DamMoreActions.tsx

```diff
@@ -26,6 +30,49 @@ export const DamMoreActions = ({ button, transformOrigin, anchorOrigin }: DamMor
         setAnchorEl(null);
     };
 
+    const selectionMapValues = Array.from(damSelectionActionsApi.selectionMap.values());
+    const lengthOfSelectedFiles = selectionMapValues.filter((value) => value === "file").length;
+    const isFolderInSelection = selectionMapValues.some((value) => value === "folder");
+    const onlyFoldersSelected = selectionMapValues.every((value) => value === "folder");
+
+    const snackbarComponent = (
+        <Snackbar
+            anchorOrigin={{ vertical: "bottom", horizontal: "left" }}
+            autoHideDuration={5000}
+            TransitionComponent={(props: SlideProps) => <Slide {...props} direction="right" />}
+            message={intl.formatMessage({
+                id: "comet.dam.moreActions.folderNotDownloaded",
+                defaultMessage: "Download of files started successfully. Folder downloads are not supported yet.",
+            })}
+        />
+    );
+
+    const onDownloadClick = () => {
```

Comment:
Callback handlers are usually named like `handle...`, e.g., `handleDownloadClick`. Please rename. See docs for reference: https://react.dev/learn/responding-to-events